zipline/data/loader.py

```diff
@@ -50,6 +51,7 @@ def last_modified_time(path):
     """
     Get the last modified time of path as a Timestamp.
     """
+    ensure_file(path)
```

Comment:
Oh, I think we don't want to do this, since it basically subverts the check below in production code. This function no longer just returns the last modified time, it first makes it "now". I think where we were copying over the market data previously (for tests only), we want to replace that with touching the market data.